docs/design/StratisSoftwareDesign.lyx

```diff
@@ -1001,6 +1001,21 @@ Most stratisd D-Bus methods require root permissions.
 Prevent Spoofing
 \end_layout
 
+\begin_layout Subsection
+Querying stratisd state via D-Bus
+\end_layout
+
+\begin_layout Standard
+Instead of providing overly verbose stratisd internal state logging, stratisd
```

Comment:
Maybe ```stratisd exposes a reporting interface that allows user to query the state of stratisd's internal data structures for debugging or error reporting.``` instead. You could skip mentioning the previous state dumps, they are so obviously a bad idea that it seems like there's no real need to justify this current approach in contrast to the previous.